### DISCUSSION TITLE
"metadata" folder deprecation?

### DISCUSSION THREAD

**Discussion by BrianCrowe3** [jellyfin/jellyfin#12029] [Q&A]
After updating to 10.9.6 all episode thumbnails for newly added shows and episodes are being saved in the show's root directory instead of the "metadata" folder like jellyfin was doing for years. I'm assuming this is expected behavior and not a bug..... But what am I supposed to do for the hundreds of shows that have episode thumbnails in "metadata" folders? Delete them and reacquire them? Is there some sort of migration script that I can use to pull the thumbnails out of the metadata folders automatically?

This seems like a weird change to randomly make after all these years.

**Comment by Shadowghost** [comment]
We are now scraping both while preferring saving as `thumb` next to the episode like any other media management software does it too.
The reason why this was changed in #11934 is that we did never scrape the images saved within the metadata directories - essentially rendering them useless. Now we do that but save to the more standard location.
If you want to migrate, a scan with replace images should do just fine. Keep in mind though that any custom images would be lost and overridden by the ones provided by the metadata providers.

**Reply by BrianCrowe3** [reply]
> We are now scraping both while preferring savin as `thumb` next to the episode like any other media management software.

Does the image need to have "-thumb" attached to the end of it? I've been messing around with jellyfin all day today, and it seems like giving the image the exact same filename as the episode and putting it in the same directory used to work, but now it doesn't.

And I know there's probably not much demand for it, but would it be possible to offer a library setting to choose where episode thumbnails are saved? I rather liked the thumbnails being saved into the metadata folder, though I definitely see why you'd want to standardize with other media management software.

**Reply by Shadowghost** [reply]
> Does the image need to have "-thumb" attached to the end of it? I've been messing around with jellyfin all day today, and it seems like giving the image the exact same filename as the episode and putting it in the same directory used to work, but now it doesn't.

That's actually a bug fixed by #12039
